opentelemetry-sdk/src/opentelemetry/sdk/_configuration/__init__.py

```diff
@@ -55,26 +52,88 @@
 
 _EXPORTER_OTLP = "otlp"
 _EXPORTER_OTLP_PROTO_GRPC = "otlp_proto_grpc"
+_EXPORTER_OTLP_PROTO_HTTP = "otlp_proto_http"
 
 _RANDOM_ID_GENERATOR = "random"
 _DEFAULT_ID_GENERATOR = _RANDOM_ID_GENERATOR
 
+_logger = logging.getLogger(__name__)
+
 
 def _get_id_generator() -> str:
     return environ.get(OTEL_PYTHON_ID_GENERATOR, _DEFAULT_ID_GENERATOR)
 
 
-def _get_exporter_names(names: str) -> Sequence[str]:
-    exporters = set()
+def _get_otlp_exporter_entry_point(
+    otlp_protocol: Optional[str],
+) -> Literal[_EXPORTER_OTLP_PROTO_GRPC, _EXPORTER_OTLP_PROTO_HTTP]:
+    if otlp_protocol == "grpc":
+        return _EXPORTER_OTLP_PROTO_GRPC
+    if otlp_protocol == "http/protobuf":
+        return _EXPORTER_OTLP_PROTO_HTTP
+    raise RuntimeError(
+        f"Unsupported OTLP protocol '{otlp_protocol}' is configured"
+    )
+
+
+def _get_exporter_entry_point(
+    exporter_name: str, telemetry_type: Literal["traces", "metrics", "logs"]
+):
+    if exporter_name not in (
+        _EXPORTER_OTLP,
+        _EXPORTER_OTLP_PROTO_GRPC,
+        _EXPORTER_OTLP_PROTO_HTTP,
+    ):
+        return exporter_name
+
+    exporter_entry_point = None
+
+    # Checking env vars for OTLP protocol (grpc/http).
+    # If grpc/http already specified by exporter_name, will only log a warning
+    # in case of a conflict.
+    otlp_protocol_var = environ.get(
+        f"OTEL_EXPORTER_OTLP_{telemetry_type.upper()}_PROTOCOL"
```

Comment:
feels a bit risky that this uses a formatted literal but other places uses the consts (we will have to remember to update here as well if the consts changes) but I am not sure I see an easier solution. Maybe a mapping between the `telemetry_type` literals to the consts? a bit more verbose but a change to the const value won't break it.